x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/helpers.test.tsx

```diff
@@ -300,28 +300,28 @@ describe('rule helpers', () => {
   });
 
```

Comment:
can you add tests covering the new behavior where seconds > 60 or minutes >60 are still returned as seconds or minutes respectively?